### ISSUE TITLE
Fix flakey UT nebula_store_test

### ISSUE THREAD

**Issue by dangleptr** [vesoft-inc/nebula#707]
http://ci.vesoft.com/job/Nebula-Graph-UnitTest/1597/consoleFull

**Comment by dangleptr** [comment]
```
19:36:19 42: [       OK ] NebulaStoreTest.SimpleTest (1545 ms)
19:36:19 42: [ RUN      ] NebulaStoreTest.PartsTest
19:36:19 42: I0731 19:36:19.837786 122023 RocksEngine.cpp:106] open rocksdb on /tmp/nebula_store_test.4MT3kb/disk1/nebula/0/data
19:36:19 42: I0731 19:36:19.947345 122023 RocksEngine.h:104] Release rocksdb on /tmp/nebula_store_test.4MT3kb/disk1/nebula/0
19:36:19 42: I0731 19:36:19.947736 122023 RocksEngine.cpp:106] open rocksdb on /tmp/nebula_store_test.4MT3kb/disk2/nebula/0/data
19:36:20 42: I0731 19:36:20.045956 122023 RocksEngine.h:104] Release rocksdb on /tmp/nebula_store_test.4MT3kb/disk2/nebula/0
19:36:20 42: I0731 19:36:20.046301 122023 NebulaStore.cpp:51] Start the raft service...
19:36:20 42: I0731 19:36:20.046525 122023 RaftexService.cpp:65] Init thrift server for raft service.
19:36:20 42: I0731 19:36:20.046612 122151 RaftexService.cpp:92] Starting the Raftex Service
19:36:20 42: E0731 19:36:20.048321 122151 ThriftServer.cpp:440] Got an exception while setting up the server: No threads available
19:36:20 42: E0731 19:36:20.048389 122151 RaftexService.cpp:83] Setup the Raftex Service failed, error: No threads available
19:36:20 42: I0731 19:36:20.049808 122164 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7d300 in thread 0x7f558788f000
19:36:20 42: I0731 19:36:20.049891 122164 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7df00 in thread 0x7f558788f000
19:36:20 42: I0731 19:36:20.049966 122165 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7d600 in thread 0x7f5586cad280
19:36:20 42: I0731 19:36:20.050034 122165 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7e200 in thread 0x7f5586cad280
19:36:20 42: I0731 19:36:20.050122 122166 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7d900 in thread 0x7f5586380000
19:36:20 42: I0731 19:36:20.050199 122166 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7e500 in thread 0x7f5586380000
19:36:20 42: I0731 19:36:20.050307 122167 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7dc00 in thread 0x7f558606e000
19:36:20 42: I0731 19:36:20.050401 122167 Acceptor.cpp:453] Dropping all connections from Acceptor=0x7f5586c7e800 in thread 0x7f558606e000
19:36:20 42: I0731 19:36:20.050597 122023 RaftexService.cpp:143] Server thread has stopped. Service on port 0 is ready to be destroyed
19:36:20 42: E0731 19:36:20.050616 122023 NebulaStore.cpp:56] Start the raft service failed
19:36:20 42: *** Aborted at 1564572980 (unix time) try "date -d @1564572980" if you are using GNU date ***
19:36:20 42: PC: @          0x11d5a52 std::vector<>::operator[]()
19:36:20 42: *** SIGSEGV (@0x38) received by PID 122023 (TID 0x7f558d012980) from PID 56; stack trace: ***
19:36:20 42:     @     0x7f558c2bd6cf (unknown)
19:36:20 42:     @          0x11d5a52 std::vector<>::operator[]()
19:36:20 42:     @          0x11c7ebd _ZZN6nebula7kvstore30NebulaStoreTest_PartsTest_Test8TestBodyEvENKUliE_clEi
19:36:20 42:     @          0x11c85ba nebula::kvstore::NebulaStoreTest_PartsTest_Test::TestBody()
19:36:20 42:     @          0x19e4299 testing::internal::HandleExceptionsInMethodIfSupported<>()
19:36:20 42:     @          0x19dbb99 testing::Test::Run()
19:36:20 42:     @          0x19dbc7e testing::TestInfo::Run()
19:36:20 42:     @          0x19dbdb4 testing::TestCase::Run()
19:36:20 42:     @          0x19dc277 testing::internal::UnitTestImpl::RunAllTests()
19:36:20 42:     @          0x19e4719 testing::internal::HandleExceptionsInMethodIfSupported<>()
19:36:20 42:     @          0x19dc3ae testing::UnitTest::Run()
19:36:20 42:     @          0x11d04f7 RUN_ALL_TESTS()
19:36:20 42:     @          0x11ccae2 main
19:36:20 42:     @     0x7f558bf03444 __libc_start_main
19:36:20 42:     @          0x11c533e (unknown)
19:36:20 42/72 Test #42: nebula_store_test ................***Exception: SegFault  3.20 sec
```

**Comment by dangleptr** [comment]
Fixed by #727
